runtime.md

```diff
@@ -90,18 +94,27 @@ Attempting to start an already started container MUST have no effect on the cont
 `run <container-id>`
 
 This operation MUST generate an error if it is not provided the container ID.
+Attempting to run a container that does not exist MUST generate an error.
 This operation MUST invoke the `create` operation, and if there are no errors, then invoke the `start` operation.
 The implementation MAY immediately execute `start` without any pause after `create` completes.
 For example if the implementation used a temporary process to implement `create`, it need not do this in the implementation of `run`.
 
+### Stop
+`stop <container-id>`
+
+This operation MUST generate an error if it is not provided the container ID.
+Attempting to stop a container that is not running MUST have no effect on the container and MUST generate an error.
+This operation MUST stop all of the processes running within the scope of the container.
```

Comment:
On Mon, Apr 25, 2016 at 01:03:09PM -0700, Doug Davis wrote:
> > +This operation MUST stop all of the processes running within the scope of the container.
> 
> I tweaked the sentence to avoid this issue :-)   will push soon

6ae7c16d → 1aae8aa0 dropped the “scope” language, but I still think
“processes running within the container” isn't well defined.